backend/settings.py

```diff
@@ -0,0 +1,4 @@
+BEER_PAGE_LIMIT = 10
```

Comment:
instead of having `os.getenv(..)` when each of these is called, you can define each setting here like:
`BEER_PAGE_LIMIT=int(os.getenv('BEER_PAGE_LIMIT','100'))`

this way you only read once (not on every usage) the environment variable